css-positioning/fixed.css

```diff
@@ -6,7 +6,8 @@ html {
   font-size: 3rem;
   text-align: center;
   position: fixed;
-  padding-left: 400px
+  top: 100px;
+  left:450px;
```

Comment:
Rather than this manual movement, try making this element the full width of the viewport. It already has `text-align: center` applied, which will handle centering the text.

Reply:
![Screen Shot 2021-01-23 at 2 44 30 PM](https://user-images.githubusercontent.com/69738204/105612391-dfb4c180-5d89-11eb-8e2a-2e7b571bb156.png)